### ISSUE TITLE
[Bug]: F-Droid can't build

### ISSUE THREAD

**Issue by licaon-kter** [simplex-chat/simplex-chat#3550]
ref https://gitlab.com/fdroid/fdroiddata/-/jobs/5739641334#L1861

https://github.com/simplex-chat/simplex-chat/blob/v5.4.1/scripts/nix/sha256map.nix#L5

vs

https://github.com/simplex-chat/simplex-chat/blob/v5.4.1-fdroid/scripts/nix/sha256map.nix#L5

/LE: fyi https://gitlab.com/fdroid/fdroiddata/-/commit/b205c7396e02416e08ce5a951639604cdfacd10c

**Comment by epoberezkin** [comment]
right, thank you! should I fix, remove the tag and create it again - will it start rebuilding?

I actually spotted this change today and was a bit puzzled how did it happen and why it was building in our nix... Possibly, because of cache?

**Comment by licaon-kter** [comment]
> will it start rebuilding?

No, as the `commit:` points to a commit hash, and that needs to be updated now.

**Comment by epoberezkin** [comment]
the root cause was this: https://github.com/simplex-chat/simplex-chat/commit/4c6387c8540531b7e157e577568a3ef96204f16d

This script was correct in stable, incorrect in stable-android branch

**Comment by linsui** [comment]
https://monitor.f-droid.org/builds/log/chat.simplex.app/238#site-footer It failed again. Could you please take a look?

**Comment by licaon-kter** [comment]
While this looks like a network glitch, the question is, why is it downloading binaries from other nix caches in the first place? Or am I misunderstanding things?
